src/processes.c

```diff
@@ -211,12 +213,16 @@ typedef struct procstat
 	derive_t io_syscr;
 	derive_t io_syscw;
 
+	unsigned long nfd;
+
 	struct procstat   *next;
 	struct procstat_entry_s *instances;
 } procstat_t;
 
 static procstat_t *list_head_g = NULL;
 
+static _Bool ps_fd = 0;
```

Comment:
The use of this variable was not obvious to me. Can you please rename to `collect_nfd`? We use `collect_*` in many other plugins for variables of this purpose.